spanner/api/Spanner.Samples/WriteDatatypesDataAsync.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_insert_datatypes_data]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public class WriteDatatypesDataAsyncSample
+{
+    public async Task WriteDatatypesDataAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        byte[] exampleBytes1 = Encoding.UTF8.GetBytes("Hello World 1");
+        byte[] exampleBytes2 = Encoding.UTF8.GetBytes("Hello World 2");
+        byte[] exampleBytes3 = Encoding.UTF8.GetBytes("Hello World 3");
+
+        var availableDates1 = new List<DateTime>();
+        availableDates1.InsertRange(0, new DateTime[] {
+                DateTime.Parse("2020-12-01"),
```

Comment:
This can fail or parse to an unexpected result depending on the Culture set in the environment where it's run.

Reply:
```suggestion
                new DateTime(2020, 12, 01, 16, 32, 0, DateTimeKind.Utc),
```
And same with the rest. Parsing literal strings is not the best way to create dates in C#